unit-tests/py/common.py

```diff
@@ -0,0 +1,117 @@
+import os, sys, subprocess, traceback
+import pyrealsense2 as rs
+
+n_assertions = 0
+n_failed_assertions = 0
+n_tests = 0
+n_failed_tests = 0
+test_failed = False
+
+# If this is the first time running this script we set the wanted enviroment, However it is impossible to change the current running 
+# environment so we rerun the script in a child process that inherits the enviroment we set
+def set_env_vars(env_vars):
+    if len(sys.argv) < 2:
+        for env_var, val in env_vars.items():
+            os.environ[env_var] = val
+        sys.argv.append("rerun")
+        cmd = ["py", "-3"] + sys.argv
+        p = subprocess.run( cmd, stderr=subprocess.PIPE, universal_newlines=True )
+        exit(p.returncode)
+
+# Returns the first device that was found, if no device is found the test is aborted
```

Comment:
Make it clear: `aborted` suggests failure, but this is a success. Explain why we do this.